### PR TITLE
MalSearch Plugin

### PR DESCRIPTION
**Pull request by DiekoMA**

### CODE REVIEW

**Comment by taooceros** [comment]
The website is not available?

**Review by MayowaAk — COMMENTED** [review]
Fixed the mistyped url so the site should load now

File: plugins.json [hunk 1]
```diff
@@ -882,5 +882,18 @@
...
+        "ActionKeyword": "mal",
+        "Name": "MalSearch",
+        "Description": "Search for anime and manga",
+        "Author": "MayowaA",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/MayowaAk/Flow.Launcher.Plugin.AnimeSearch",
+        "UrlDownload": "https://github.com/MayowaAk/Flow.Launcher.Plugin.AnimeSearch/releases/download/Testing/MalSearch.zip",
+        "UrlSourceCode": "https://github.com/MayowaAk/Flow.Launcher.Plugin.AnimeSearch/tree/master/",
+        "IcoPath": "icon.png",
```
Comment (taooceros) [inline plugins.json:897]:
```suggestion
        "IcoPath": "[icon.png](https://cdn.jsdelivr.net/gh/MayowaAk/Flow.Launcher.Plugin.AnimeSearch@master/AnimeSearch/Flow.Launcher.Plugin.AnimeSearch/MyAnimeList_Logo.png)",
```

**Review by taooceros — CHANGES_REQUESTED** [review]

**Comment by Garulf** [comment]
JSON validator test is failing due to an extra comma.

**Review by jjw24 — CHANGES_REQUESTED** [review]
Hi @DiekoMA how's it going?

I tested your plugin but all I got was a pop up saying 'Test' when I search. Could you please update it so it's not the test release.

File: plugins.json [hunk 2]
```diff
@@ -882,5 +882,18 @@
...
+        "ActionKeyword": "mal",
+        "Name": "MalSearch",
+        "Description": "Search for anime and manga",
+        "Author": "MayowaA",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/MayowaAk/Flow.Launcher.Plugin.AnimeSearch",
+        "UrlDownload": "https://github.com/MayowaAk/Flow.Launcher.Plugin.AnimeSearch/releases/download/Testing/MalSearch.zip",
+        "UrlSourceCode": "https://github.com/MayowaAk/Flow.Launcher.Plugin.AnimeSearch/tree/master/",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/MayowaAk/Flow.Launcher.Plugin.AnimeSearch@master/AnimeSearch/Flow.Launcher.Plugin.AnimeSearch/MyAnimeList_Logo.png",
```
Comment (jjw24) [inline plugins.json:897]:
```suggestion
        "IcoPath": "https://cdn.jsdelivr.net/gh/MayowaAk/Flow.Launcher.Plugin.AnimeSearch@master/AnimeSearch/Flow.Launcher.Plugin.AnimeSearch/MyAnimeList_Logo.png"
```

**Pull request closed without merging**